### ISSUE TITLE
Yoroi to Nami import issues

### ISSUE THREAD

**Issue by cryptogod007** [Berry-Pool/nami-wallet#377]
I imported seed phrase from yoroi to nami. Yoroi wallet had 3.70 ada I left it over night. Next day opened yoroi ada gone. I see ada come into wallet then all ada plus 15 nft assets sent out to wallet address when checked on pool.pm is a goat staking pool. I should mention I did not do this transaction and also used nami app from play store to import the seed phrase but I couldn't see how it worked on my phone so switched to chrome on laptop. I can see the transactions into nami first being 79.51 ada into then out again. More ada in and out different amounts  3 assets in but where are they? I sent more ada into nami 45 ada in total you can see it come in and then right back out. Is this normal? Where are my 3 rec assets and 12 missing? Please help

**Comment by cryptogod007** [comment]
All ada received is being sent back out to goat when I checked the to wallet in pool.pm

**Comment by cryptogod007** [comment]
When I search my nami wallet address in pool.pm it returns stake1u9we02 could this be a hacked nami app from play store sending all received funds and assets to goat?

**Comment by HeptaSean** [comment]
Yes, the Android app is a scam. There is no Android app. Nami is a chrome extension and nothing else.

Yes, the target address of the scammers is delegated to GOAT, which does not mean that the stake pool has anything to do with them. Anyone can delegate to any stake pool.

Unfortunately, there is nothing anyone can do, once the transaction has been done, not you, not the Nami developers, not the people running the Cardano network.

You can try to report it to authorities, but even if they understand what has happened, the chances are not great that they can do anything if the criminals sit on the other side of the globe.

**Comment by cryptogod007** [comment]
Damn it lost 20k in nfts thanks for the information
